css/index.css

```diff
@@ -0,0 +1,313 @@
+    width: 100%;
+}
+
+.nav { 
+    float: left;
+    color: #ccd1d7;
+    font-weight: bold;
+    font-family: 'Manrope', sans-serif;
+}
+
+.patch { 
+    font-weight: bold;
+    color:#1a2939;
+}
+
+.depo {
+    color:#47a8d1;
+}
+
+.img {
+    margin-left: 550px;
+    margin-top:-20px;
+    overflow: hidden;
+}
+
+img {
+    border-radius: 25px;
+    height: 25px;
+    width: 25px;
+}
+
+
+.btn-primary {
+    background-color:#4ecb73;
+    color: #e5fdf9;
+    height: 45px;
+    width: 145px;
+    border-radius: 8px;
+    float: right;
+    margin-left: 868px;
+    font-weight: bold;
+    border: none;
+    margin-bottom: 15px
+}
+
+.btn-depo {
+    float: left;
+}
+
+.head3 {
+    color: #313e4c;
+    margin-top: 5px;
+    font-weight:normal;
+}
+
+.table {
+    text-align: left;
+}
+
+table {
+    border-collapse: collapse;
+    height: 100%;
+    width: 100%;
+}
+
+th {
+    padding: 10px 0px;
+    background-color: #f9fbfc;
+    font-size: 10px;
+    color:#9aa3b0;
+    letter-spacing: 1px;
+}
+
+td {
+    padding-top: 10px;
+    padding-left: 0px;
+}
+
+.line {
+    border-bottom: 1px solid #f4f3f6;
+}
+
+th.row1 {
+    color:#1e2b3b;
+    padding-right:200px;
+    padding-left: 8px;
+}
+    
+th.row2{
+    color:#1e2b3b;
+    padding-right: 195px;
+    padding-left: 8px;
+}
+    
+td img {
+    float: left;
+    margin-left: 2px;
+    padding: 9px;
+    height: 35px;
+    width: 35px;
+}
+    
+.round-img {
+    height: 25px;
+    width: 25px;
+}
+
+.round-img-over{
+    height: 25px;
+    width: 25px;
+    left: 0px;
+    position: relative;
+    
+}
+
+.round-img-over2{
+        height: 25px;
+        width: 25px;
+        left: -25px;
+       position: relative;
+        
+}
+
+span {
+    font-size: 9px;
+    color: #abb1bb;
+}
+
+.name {
+    font-size: 15px;
+    color: #47a8d1;
+}
+
+h3 {
+    color:#8a96a0;
+    margin-bottom: -35px;
+    margin-top: 40px;
+}
+
+  
+.comment {
+    width: 60px;
+    height: 27px;
+    background:#fbe0f5;
+    position: relative;
+    -moz-border-radius: 15px;
+    -webkit-border-radius: 12px;
+    border-radius: 5px;
+    margin-left: 200px;
+    margin-top: -54px;
+    float: left;
+}
+
+.comment:before {
+    content: "";
+    position: absolute;
+    right: 100%;
+    top: 4px;
+    width: 0;
+    height: 0;
+    border-top: 10px solid transparent;
+    border-right: 5px solid #fbe0f5;
+    border-bottom: 10px solid transparent;
+}
+
+.comment p{
+    color:#de89ce;
+    padding-top: 0px;
+    text-align: center;
+    font-size: 12px;
+    margin-top: 5px;
+}
+
+.next {
+    color:black;
+    text-decoration: none;
+    margin-left:60px;
+    size: 30px;
+}
+
+.tab2 {
+    margin-top:50px;
+}
+
+.arrow {
+    border: solid gray;
+    border-width: 0 1px 1px 0;
+    display: inline-block;
+    padding: 3px;
+    height: 4px;
+    width: 4px;
+    float: right;
+}
+  
+.right {
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
+}
+
+.down {
```

Comment:
@indalgit - do you know what is css specificity rules ? read about specificity rules. And here you should use **.arrow.down** because as developer just down isn't self descriptive i.e not able to understand the uses of this class.